### DISCUSSION TITLE
Idea: Kaboom packager

### DISCUSSION THREAD

**Discussion by lajbel** [replit/kaboom#173] [Ideas]
Maybe you have it in mind, it would be great if you can with a single click compile kaboomjs to nw.js or electron as it has for example http://renjs.net/, it could even be integrated with the replit editor (?)

**Comment by slmjkdbtl** [comment]
Good idea.

I also started a project a while ago to run kaboom natively without any browser engine, kinda on pause right now but hopefully that works and can replace nwjs or electron (for kaboom)
